interpreters/wamr/Kconfig

```diff
@@ -92,6 +92,14 @@ config INTERPRETERS_WAMR_MINILOADER
 	---help---
 	Mini-loader don't check the integrity of wasm module
 
+config INTERPRETERS_WAMR_REF_TYPES
```

Comment:
where is the corresponding wamr patch?

Reply:
https://github.com/bytecodealliance/wasm-micro-runtime/pull/1711